Mu2eKinKal/test/TTTA.fcl

```diff
@@ -67,7 +67,7 @@ physics.producers.TTmakeSH.FilterHits : false
 physics.producers.TTmakePH.TestFlag : true
 #physics.producers.TTmakePH.FilterHits : true
 physics.producers.TTmakePH.StrawHitSelectionBits : ["EnergySelection","TimeSelection","RadiusSelection"]
-physics.producers.TTmakePH.StrawHitMask          : ["Noisy"]
+physics.producers.TTmakePH.StrawHitMask          : ["Dead","Noisy"]
```

Comment:
When simulating we have:
https://github.com/Mu2e/Offline/blob/4e3de16eacd23d3e9e8de089b8f107d1a5aaa470/TrackerMC/src/MakeStrawGasSteps_module.cc#L347
Which I think guarantees we have no steps or digis from dead straws.  Do we need this for real data, where the 'dead' channels still produce noise?  Or is there a gap in the logic?

Reply:
We could plausibly have a channel marked dead that we still included in the readout mask and could generate noise, but also this just guarantees in sim that if the dead straw condition is added at the digi or reco stage it will still be applied